algorithms/appo/learner.py

```diff
@@ -109,6 +109,16 @@ def __init__(
 
         self.process = Process(target=self._run, daemon=True)
 
+        if is_continuous_action_space(self.action_space):
+            raise NotImplementedError(f'Continuous Action spaces not supported!')
```

Comment:
Why is this not supported? I think this should only throw when we use KL loss?

Reply:
And the exception message should be something like:
"KL-divergence exploration loss is not supported with continuous action spaces. Use entropy exploration loss"

Reply:
Oh damn, my mistake, I'll fix it